demos/src/news-package.mustache

```diff
@@ -0,0 +1,33 @@
+<div class="o-topper o-topper--news-package o-topper--color-wheat">
+	<div class="o-topper__content">
+		<div class="o-topper__tags">
+			<div class="o-topper__primary-theme">
```

Comment:
Is `o-topper__primary-theme` still needed? I was thinking "theme" is confusing here but can't see any CSS attached to this, so it can be deleted?

Reply:
i'll make sure it wasn't being used anywhere in next-article

Reply:
only in the print styles, i'll remove it and use `o-topper__tags` in that case

Reply:
- [ ] remove `o-topper__primary-theme` from demos